data/decks/linne.json

```diff
@@ -0,0 +1,190 @@
+{
+	"id": "linne",
+	"season": 6,
+	"display_name": "Linne",
+	"cardback": "cardback_linne.jpg",
+	"exceed_cost": 1,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "draw",
+			"amount": 2,
+			"and": {
+				"effect_type": "draw",
+				"opponent": true,
+				"amount": 1,
+				"and": {
+					"effect_type": "choice",
+					"choice": [
+						{ "effect_type": "advance", "amount": 1 },
+						{ "effect_type": "retreat", "amount": 1 }
+					]
+				}
+			}
+		}
+	}],
+	"character_action_exceeded": [
+		{
+			"gauge_cost": 0,
+			"force_cost": 0,
+			"can_continuous_boost_from_gauge": true,
+			"effect": {
+				"effect_type": "boost_from_gauge",
+				"limitation": "continuous"
+			}
+		},
+		{
+			"gauge_cost": 0,
+			"force_cost": 0,
+			"min_hand_size": 1,
+			"action_name": "Revert",
+			"effect": {
+				"effect_type": "gauge_from_hand",
+				"min_amount": 1,
+				"max_amount": 1,
+				"and": {
+					"effect_type": "revert",
+					"and": {
+						"effect_type": "take_bonus_actions",
+						"amount": 1
+					}
+				}
+			}
+		}
+	],
+	"ability_effects": [
+	],
+	"exceed_ability_effects": [
+		{
+			"timing": "set_strike",
+			"condition": "used_character_action",
+			"condition_details": ["exceed", 0],
+			"character_effect": true,
+			"effect_type": "powerup",
+			"amount": 1
+		}
+	],
+	"cards": [
+		{
+			"definition_id": "linne_divineblaze",
+			"image": "linne_divineblaze.tres"
+		},
+        {
```

Comment:
tabs v spaces?

Reply:
Sorry, not sure why my editor is doing this...